### PR TITLE
Update README.md (issue #1305 )

### PR DESCRIPTION
**Pull request by prachi237**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

**Review by ForgottenProgramme — COMMENTED** [review]
LGTM! :)

**Review by viniciusdc — COMMENTED** [review]
Hi @prachi237 it's really good, thanks for the PR. May you add a brief comment explaining that the PR must only change the files inside `.src` and **not** the ones inside the '.docs' ? :smile:

**Comment by prachi237** [comment]
> Hi @prachi237 it's really good, thanks for the PR. May you add a brief comment explaining that the PR must only change the files inside `.src` and **not** the ones inside the '.docs' ? 😄

@viniciusdc  sir that is already mentioned 🤔

File: README.md [hunk 1]
```diff
@@ -11,15 +11,18 @@ If you have questions or need help, please check out our documentation for a [li
...
-5. `cd newsfeed && pip install --no-deps .`
-6. `cd ../src`
-7. `make html`
+      *  `conda env create -f ./.ci_scripts/environment.yml`
+      *  `conda activate conda-forge-docs`
+      *  `cd newsfeed && pip install --no-deps .`
+      *  `cd ../src`
+      *  `make html`
+4.  Make and commit your changes.
+5.  Submit a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) to the main repository proposing your changes.
```
Comment (ForgottenProgramme) [inline README.md:26]: Hey @prachi237, what @viniciusdc is saying is that we should add a small note at the end here.
"Note: All changes must be made in the /src folder only and NOT in the /docs folder."

Comment (prachi237) [reply README.md:26]: > Hey @prachi237, what @viniciusdc is saying is that we should add a small note at the end here.
> "Note: All changes must be made in the /src folder only and NOT in the /docs folder."

ahhh well got it

**Comment by prachi237** [comment]
@viniciusdc   done

File: README.md [hunk 2]
```diff
@@ -11,15 +11,20 @@ If you have questions or need help, please check out our documentation for a [li
...
+
+**Note: "All changes must be made in the /src folder and NOT in the /docs folder."**
 
 ## Improving the docs
```
Comment (viniciusdc) [inline README.md:29]: I don't see a reason for this next section anymore, could you try and assimilate this content into the beginning of your section?

**Review by viniciusdc — COMMENTED** [review]
Looks nice!

File: README.md (hunk 2 above)
Comment (prachi237) [reply README.md:29]: > I don't see a reason for this next section anymore (Improving the docs), could you try and assimilate this content into the beginning of your section? Then rename buiding the docs to Improving the docs

yes that sounds great and will try to  elaborate in a better way

**Review by viniciusdc — APPROVED** [review]
wonderful

**Pull request merged**